tests/expressions/tests.py

```diff
@@ -1357,6 +1357,40 @@ def test_raise_empty_expressionlist(self):
             ExpressionList()
 
 
+class FLookupTests(TestCase):
```

Comment:
Is the class name correct? I don't see any F expressions -- maybe I misunderstood what "FLookup" means.

Reply:
This was from jarshwah's initial hint in the ticket. I agree, your suggested name is better.